### PR TITLE
Release 0.2.2: Bug Fixes

### PR DESCRIPTION
**Pull request by ax3l**
This is a bugfix release for the latest PIConGPU release.

During backport I modified the icc fixes so this release still keeps C++98 compatibility for the `0.2.X` line.

## Related Project

see [0.2.2 Backports](https://github.com/ComputationalRadiationPhysics/picongpu/projects/6)

## Tests

I did a CUDA 8.0 (C++11) and CUDA 7.0 (C++98) compile test on hypnos-hzdr.

## Version Bump

Updated `src/picongpu/include/version.hpp` to `0.2.2`.

## Changelog

Laser wavepacket, vacuum openPMD & icc     

This release fixes small a broken laser profile (wavepacket), allows to use icc as the host compiler, fixes a bug when writing openPMD files in simulations without particle species ("vacuum") and a problem with GPU device selection shared node usage via `CUDA_VISIBLE_DEVICES`.

### Changes to "0.2.1"

**Bug Fixes:**
 - add missing minus sign wavepacket laser transversal #1722
 - write openMPD meta data without species #1718
 - Device Selection: Guard Valid Range #1665
 - PMacc icc compatibility:
   - `MapTuple` #1648
   - `AllCombinations` #1646

**Misc:**
 - refactor `InheritLinearly` #1647

Thank you to @psychocoderHPC and @PrometheusPi for spotting the issues and providing fixes!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.2.2
+-----
+**Date:** 2017-01-03
+
+Laser wavepacket, vacuum openPMD & icc
+
+This release fixes small a broken laser profile (wavepacket),
```
Comment (PrometheusPi) [inline CHANGELOG.md]: fixes ~small~ a broken laser profile

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+allows to use icc as the host compiler, fixes a bug when writing openPMD
+files in simulations without particle species ("vacuum") and a problem
+with GPU device selection shared node usage via `CUDA_VISIBLE_DEVICES`.
```
Comment (PrometheusPi) [inline CHANGELOG.md]: GPU device selection **during** shared node usage via

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+
+### Changes to "0.2.1"
+
+**Bug Fixes:**
+ - add missing minus sign wavepacket laser transversal #1722
+ - write openMPD meta data without species #1718
```
Comment (PrometheusPi) [inline CHANGELOG.md]: ~openMPD~ -> open**PM**D

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,34 @@
...
+ - Device Selection: Guard Valid Range #1665
```
Comment (PrometheusPi) [inline CHANGELOG.md]: why capital letters - please use lower case letters as anywhere else

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,34 @@
...
+ - PMacc icc compatibility:
+   - `MapTuple` #1648
+   - `AllCombinations` #1646
+
+**Misc:**
+ - refactor `InheritLinearly` #1647
+
+Thank you to René Widera and Richard Pausch for spotting the issues and
```
Comment (PrometheusPi) [inline CHANGELOG.md]: ~Thank you~ to -> **Thanks** to

File: CHANGELOG.md (hunk 2 above)
Comment (ax3l) [inline CHANGELOG.md]: thx, will use *on* shared node usage

File: CHANGELOG.md (hunk 1 above)
Comment (ax3l) [inline CHANGELOG.md]: thx

File: CHANGELOG.md (hunk 3 above)
Comment (ax3l) [inline CHANGELOG.md]: copied from PR title, will fix

File: CHANGELOG.md (hunk 4 above)
Comment (ax3l) [inline CHANGELOG.md]: copied from PR description... will fix

File: CHANGELOG.md (hunk 5 above)
Comment (ax3l) [inline CHANGELOG.md]: k

**Pull request merged**